### ISSUE TITLE
GUI : FED : Table accessibility violations

### ISSUE THREAD

**Issue by dixsie** [ibm-openbmc/dev#2841]
## Overview
- [ ] Tables do not have rows tabindex set to 0, causing DAP violations
- [ ] Determine whether to override or add tabindex

## Notes
- [ ] https://bootstrap-vue.org/docs/components/table#accessibility

### Browser Tests (Chrome, Firefox, Safari (Mac), Edge (Windows))
- [ ] Tested responsive layout on supported viewport sizes
- [ ] Verified functionality works as expected, e.g. success and failure use cases
- [ ] Resolved any JavaScript errors thrown to the console

### Accessibility Tests
- [ ] Verified markup is semantic
- [ ] No reported issues in IBM Accessibility Checker (Free open source tool) https://www.ibm.com/able/toolkit/tools/

### Test Hooks
- [ ] Verify need for `data-test-id` hooks with test team

**Comment by dixsie** [comment]
DAP violations are caused by bootsrap vue implementation. @derick-montague opened an issue with bootstrap vue for multiselect issues. 

Closing this story because Bootstrap vue adds `role="rowgroup"` to `thead` which causes the tabindex issue.
